at_onboarding_flutter/lib/screens/pair_atsign.dart

```diff
@@ -262,8 +261,11 @@ class _PairAtsignWidgetState extends State<PairAtsignWidget> {
         loading = true;
       });
       for (PlatformFile pickedFile in result?.files ?? <PlatformFile>[]) {
-        String? path = pickedFile.path.toString();
-        File selectedFile = File(path.toString());
+        String? path = pickedFile.path;
+        if (path == null) {
+          throw const FileSystemException('FilePicker.pickFiles returned a null path');
+        }
+        File selectedFile = File(path);
```

Comment:
File does not accept null and path is a nullable string.

Reply:
Nope if the path is null it won't go to the next step. `File(path)` will be non-nullable value as `path == null` doesn't pass.